tools/c7n_mailer/c7n_mailer/ldap_lookup.py

```diff
@@ -14,7 +14,12 @@
 import json
 import redis
 import re
-import sqlite3
+try:
+    import sqlite3
```

Comment:
oh? did sqlite3 get dropped in py3?

Reply:
I think it's a `.so` dependency that we're not necessarily uploading? Available locally but not on Lambda? Can track down further if need be ...

Reply:
https://github.com/whit537/cloud-custodian/pull/37#issuecomment-315397532 &rarr; https://stackoverflow.com/questions/44058239/

Reply:
Comment added in 44fb6236250ad01721b30f8834c642eaba939505.

Reply:
Actually, scratch that: link is already in error message! 😁

https://github.com/capitalone/cloud-custodian/blob/fd5d5c500676bf0a05ad0ae1ece57c445ef8b441/tools/c7n_mailer/c7n_mailer/ldap_lookup.py#L47-L49